test/zero-downtime/sequences_test.go

```diff
@@ -0,0 +1,204 @@
+package zero_downtime
+
+import (
+	"fmt"
+	"github.com/benbjohnson/clock"
+	"github.com/keptn/go-utils/pkg/api/models"
+	"github.com/keptn/go-utils/pkg/lib/keptn"
+	"github.com/keptn/go-utils/pkg/lib/v0_2_0"
+	testutils "github.com/keptn/keptn/test/go-tests"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/suite"
+	"gopkg.in/yaml.v3"
+	"sync"
+	"sync/atomic"
+	"testing"
+	"time"
+)
+
+type TestSuiteSequences struct {
+	suite.Suite
+	env     *ZeroDowntimeEnv
+	project string
+}
+
+type TriggeredSequence struct {
+	keptnContext string
+	projectName  string
+	sequenceName string
+}
+
+func NewTriggeredSequence(keptnContext string, projectName string, seqName string) TriggeredSequence {
+	return TriggeredSequence{
+		keptnContext: keptnContext,
+		projectName:  projectName,
+		sequenceName: seqName,
+	}
+}
+
+func (suite *TestSuiteSequences) SetupSuite() {
+
+	suite.T().Log("Starting test for sequences")
+	// if needed the following line can setup a project eat very clock tick
+	suite.createNew()
+}
+
+func (suite *TestSuiteSequences) createNew() {
+	var err error
+	projectName := "zd-sequence" + suite.env.gedId()
+	suite.T().Logf("Creating project with id %s ", projectName)
+	suite.project, err = testutils.CreateProject(projectName, suite.env.ShipyardFile)
+	suite.Nil(err)
+	output, err := testutils.ExecuteCommand(fmt.Sprintf("keptn create service %s --project=%s", "myservice", suite.project))
+	suite.Nil(err)
+	suite.Contains(output, "created successfully")
+	suite.T().Logf("Starting test for project %s ", suite.project)
+}
+
+func (suite *TestSuiteSequences) BeforeTest(suiteName, testName string) {
+	atomic.AddUint64(&suite.env.FiredSequences, 1)
+	suite.T().Log("Running one more test, tot ", suite.env.FiredSequences)
+}
+
+//Test_Sequences can be used to test a single run of the sequence test suite
+func Test_Sequences(t *testing.T) {
+	Env := setEnv(t)
+
+	s := &TestSuiteSequences{
+		env: Env,
+	}
+	suite.Run(t, s)
+}
+
+func setEnv(t *testing.T) *ZeroDowntimeEnv {
```

Comment:
```suggestion
func setSequencesTestEnv(t *testing.T) *ZeroDowntimeEnv {
```

It can be misunderstood, because `os.SetEnv()` is a `os` library method and we shouldn't use the same naming